src/formats/roam-json.ts

```diff
@@ -86,7 +112,41 @@ export class RoamJSONImporter extends FormatImporter {
 				}
 				const filename = `${graphFolder}/${pageName}.md`;
 
-				const markdownOutput = await this.jsonToMarkdown(graphFolder, attachmentsFolder, pageData);
+				// if title option is enabled
+				let YAMLtitle: string;
+				if (this.titleYAML === true) {
+					YAMLtitle = pageData.title;
+				} else {
+					YAMLtitle = '';
+				}
+				// if timestamp option is enabled
+				// set up numbers to pass
+				let pageCreateTimestamp: number;
```

Comment:
Can we extract these timestamps in `jsonToMarkdown` so we don't need to pass them as parameters?

Reply:
I wasn't able to, that's how I initially tried to get the dates. It didn't allow for the page dates to influence the timestamps, just each block-level. That ended up causing an issue where the `created` date could show as the oldest block's date, even if the page was created previous to that without a block-level beforehand (for example, the Daily Note being created automatically).

If there's a way to get that, please let me know. I'd rather try to do everything under `jsonToMarkdown` without needing to add parameters